tests/src/JIT/Performance/CodeQuality/BenchmarksGame/spectralnorm/spectralnorm.cs

```diff
@@ -49,13 +50,14 @@ public static void Bench()
                     a += s.Approximate(n);
                 }
             }
-        }
-        double norm = a / (n * Iterations);
-        double expected = 1.274219991;
-        bool valid = Math.Abs(norm - expected) < 1e-4;
-        if (!valid)
-        {
-            throw new Exception("Benchmark failed to validate");
+
+            double norm = a / Iterations;
+            double expected = 1.274219991;
+            bool valid = Math.Abs(norm - expected) < 1e-4;
+            if (!valid)
+            {
+                throw new Exception("Benchmark failed to validate");
+            }
```

Comment:
I don't see any indentation issues (try looking at the full diff, since I moved some code from outside a block to inside). If you still see one let me know which line....

I was able to repro the failure locally, and have verified it is indeed fixed.